test/integration/mock/website/sitemapindex.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<sitemapindex xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+   <sitemap>
+      <loc>http://localhost:8090/sitemap.xml</loc>
+   </sitemap>
```

Comment:
Can you add a second sitemap and update the tests to check that it runs correctly.
This will require some additional pages to the `mock/website` which the second sitemap can point to